### PR TITLE
Merge gamepad and rover tutorials

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tutorials/control/gamepad.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Drive a Rover (like SCUTTLE) Using a Gamepad with a Dongle"
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:2]: The dongle erasure 🤣

File: docs/tutorials/control/gamepad.md [hunk 2]
```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
...
-{{% /alert %}}
-
 ## Requirements
 
 You will need the following hardware to complete this tutorial:
 
 - A wheeled rover, configured with a [base component](/components/base/) on the [Viam app](https://app.viam.com/).
-  This tutorial uses a [SCUTTLE rover](https://www.scuttlerobot.org/shop/) as an example but you can complete this tutorial using a different rover.
-  - Regardless of the type of base you are using, [Setting up a SCUTTLE with Viam](/tutorials/configure/scuttlebot/) is a good place to start if you haven't already configured your base.
+  This tutorial uses a [SCUTTLE rover](https://www.scuttlerobot.org/shop/) as an example but you can complete this tutorial using [Yahboom 4WD Smart Robot](https://category.yahboom.net/collections/robotics/products/4wdrobot) or an entirely different rover.
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:34]:
```suggestion
  This tutorial uses a [SCUTTLE rover](https://www.scuttlerobot.org/shop/) as an example but you can complete this tutorial using a [Yahboom 4WD Smart Robot](https://category.yahboom.net/collections/robotics/products/4wdrobot) or an entirely different rover.
```
I think

File: docs/tutorials/control/gamepad.md [hunk 3]
```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
...
 
-Plug the gamepad Bluetooth dongle into a USB port on the rover's [board](/components/board/).
-Turn on power to the rover.
+If your gamepad has a dongle, plug the gamepad Bluetooth dongle into a USB port on the rover's [board](/components/board/).
+Then turn on power to the rover.
+
+{{% expand "Click here if your gamepad does not have a dongle for bluetooth pairing instructions." %}}
+
+Make sure your bluetooth controller is in pairing mode.
+For an 8bitdo controller mode switch is set to S, hold down Start for a few seconds, and when the LED underneath the controller changes to green, press the pair button for 3 seconds.
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:50]:
```suggestion
For an 8bitdo controller, set the mode switch to **S**, hold down **Start** for a few seconds, and when the LED underneath the controller changes to green, press the pair button for 3 seconds.
```
(opt) sentence a bit hard to parse

File: docs/tutorials/control/gamepad.md [hunk 4]
```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
...
+As you do this, make sure you run this command on the machine that runs `viam-server` which may be a different machine to your laptop.
+
+This command scans all bluetooth devices.
+Find your gamepad.
+For an 8bitdo controller, the MAC address begins with E4:17:D8.
+
+![A screenshot of a Raspberry Pi terminal with the following command: sudo bluetoothctl scan on. The results of the command are displayed: a list of device MAC addresses.](/tutorials/yahboom-rover/bluetooth-scan.png)
+
+Once you find your controller, pair with the controller by running the following command: `sudo bluetoothctl pair <8bitdo-mac-address>`.
+Do not forget to take the < and > symbols out as you paste your address.
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:63]:
```suggestion
Do not forget to take the `<` and `>` symbols out as you paste your address.
```
(Opt)

File: docs/tutorials/control/gamepad.md [hunk 5]
```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
...
 {{% tab name="Config Builder" %}}
 
 Configure a [gamepad](/components/input-controller/gamepad/):
 
-- Click **Create component**.
-- Select `input_controller` for the component **Type**.
-- Select `gamepad` for the **Model**.
-- Enter `my-gamepad` as the component **Name**.
-- Click **Create**.
+Click the + icon next to Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:86]:
```suggestion
Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
```

File: docs/tutorials/control/gamepad.md [hunk 6]
```diff
@@ -23,54 +23,78 @@ date: "2022-08-10"
...
+Select the `input_controller` type, then select the `gamepad` model.
+Enter a name or use the suggested name for your input controller and click **Create**.
 
-![Blank configuration JSON](/tutorials/scuttle-gamepad/gamepad-config.png)
+![An example configuration for a linux-based gamepad input controller component in the Viam App config builder](/components/input-controller/gamepad-input-controller-ui-config.png)
+
+You can set the `auto_reconnect` attribute to `true`.
 
 {{% /tab %}}
 {{% tab name="Raw JSON" %}}
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:95]:
```suggestion
{{% tab name="JSON" %}}
```
Have been switching this as subtab name is just JSON now. Doesn't seem like a big deal either way though. Was considering also doing Config Builder - Builder but haven't done that yet, any thoughts?

File: layouts/partials/imgproc.html [hunk 7]
```diff
@@ -14,7 +14,7 @@
   {{ $imgbackup := $img.Resize .resize }}
   {{if .declaredimensions}}
 <source srcset="{{ $imgwebp.RelPermalink }}" type="image/webp" width="{{ $imgwebp.Width }}" height="{{ $imgwebp.Height }}">
-<img src="{{ $img.RelPermalink }}" width="{{ $imgwebp.Width }}" height="{{ $imgwebp.Height }}" alt="{{ .alt }}" class="{{ .class }}" style="{{ .style }}" loading="lazy">
```
Comment (sguequierre) [inline layouts/partials/imgproc.html:17]: Did you mean to commit this?

File: static/include/components/test-control/base-control.md [hunk 8]
```diff
@@ -3,3 +3,14 @@ If your base makes use of a [camera](/components/camera/) component, the **CONTR
...
 
 {{< imgproc src="/components/base/base-control-tab.png" alt="The base component in control tab" resize="350x" >}}
+
+On the **Keyboard** tab, you can toggle the keyboard control to active.
+With the **Keyboard** toggle active, use **W** and **S** to go forward and back, and **A** and **D** to arc and spin.
+
+Try driving your base around using the WASD keyboard controls.
+
+{{% alert title="Caution" color="caution" %}}
+Ensure that your robot has sufficient space to drive around without hitting anyone or anything.
```
Comment (sguequierre) [inline static/include/components/test-control/base-control.md:13]: nice

File: docs/tutorials/control/gamepad.md [hunk 9]
```diff
@@ -192,3 +216,14 @@ Each red color arrangement allows you to control the gamepad in the Viam app:
...
+## Next steps
+
+You can now drive your rover with a wireless controller.
+If you'd like to do more with your rover, check out one of these tutorials:
+
+{{< cards >}}
+{{% card link="/tutorials/get-started/try-viam-sdk/" %}}
+{{% card link="/tutorials/services/try-viam-color-detection/" %}}
+{{% card link="/tutorials/services/navigate-with-rover-base/" %}}
+{{< /cards >}}l
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:229]:
```suggestion
{{< /cards >}}
```
Snuck in there

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/tutorials/control/gamepad.md (hunk 1 above)
Comment (npentrel) [reply docs/tutorials/control/gamepad.md:2]: 🤐

File: docs/tutorials/control/gamepad.md (hunk 6 above)
Comment (npentrel) [reply docs/tutorials/control/gamepad.md:95]: Sounds good I think

File: layouts/partials/imgproc.html (hunk 7 above)
Comment (npentrel) [reply layouts/partials/imgproc.html:17]: Nope

**Review by sguequierre — APPROVED** [review]
LGTM!
